Build/Linux/DEBIAN/control

```diff
@@ -0,0 +1,14 @@
+Package: AtomicGameEngine
+Version: 0.0.8
+Installed-Size: 393064
+Section: graphics
+Maintainer: pincushion <info@atomicgameengine.com>
+Homepage: http://atomicgameengine.com
```

Comment:
pincushion is a name we used to give to novice quake players, if you'd like a real maintainer name that easily fixed too. Call it plausible denial-ability ... "blame pincushion, he b0rked it all up."